src/frontend/apps/standalone_site/src/features/Playlist/features/UpdatePlaylist/components/UpdatePlaylistPage.tsx

```diff
@@ -0,0 +1,122 @@
+import { Box, BoxProps, Button, Heading } from 'grommet';
+import { report, Spinner } from 'lib-components';
+import { PropsWithChildren, useState } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+import { useParams } from 'react-router-dom';
+
+import { ModalButtonContainer } from 'components/Modal/ModalButton';
+
+import { PlaylistForm } from '../../../components/PlaylistForm';
+import { usePlaylist } from '../api/usePlaylist';
+import { useUpdatePlaylist } from '../api/useUpdatePlaylist';
+
+import { PlaylistUserList } from './PlaylistUserList';
+
+const messages = defineMessages({
+  mainInfoTitle: {
+    defaultMessage: 'Main informations',
+    description: 'Update playlist page, main info bloc title.',
+    id: 'features.Playlist.features.UpdatePlaylist.UpdatePlaylistPage.mainInfoTitle',
+  },
+  accessTitle: {
+    defaultMessage: 'Playlist access',
+    description: 'Update playlist page, playlist access bloc title.',
+    id: 'features.Playlist.features.UpdatePlaylist.UpdatePlaylistPage.accessTitle',
+  },
+  saveButtonTitle: {
+    defaultMessage: 'Save',
+    description: 'Update playlist page, save main info button title.',
+    id: 'features.Playlist.features.UpdatePlaylist.UpdatePlaylistPage.saveButtonTitle',
+  },
+});
+
+const BlocContainer = ({ children, ...props }: PropsWithChildren<BoxProps>) => {
+  return (
+    <Box
+      round="8px"
+      background="white"
+      pad={{ vertical: 'medium', horizontal: 'large' }}
+      margin={{ top: 'medium' }}
+      elevation="even"
+      {...props}
+    >
+      {children}
+    </Box>
+  );
+};
+
+export const UpdatePlaylistPage = () => {
+  const intl = useIntl();
+  const params = useParams<{ id: string }>();
+
+  const [isEditing, setIsEditing] = useState(false);
+  const { data: playlist, isLoading } = usePlaylist(params.id);
+  const { mutate: updatePlaylist } = useUpdatePlaylist(params.id);
+
+  if (isLoading || !playlist) {
+    return <Spinner />;
+  }
+
+  return (
+    <Box gap="medium">
+      <BlocContainer>
```

Comment:
Better to keep using the component `WhiteCard` to keep consistency with the padding and margin.